pkgs/development/python-modules/returns/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, typing-extensions
+}:
+
+buildPythonPackage rec {
+  pname = "returns";
+  version = "0.19.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "4544bb67849c1ef1bbf7823759d433a773959e5b77a8fd06d01fef6d060f2ac5";
+  };
+
+  nativeBuildInputs = [
+    typing-extensions
+  ];
+
+  # Tests cannot be run because they require data that isn't included in the
+  # release tarball.
```

Comment:
If the data is in git repo but not on pypi, just use `fetchFromGitHub` instead of `fetchPypi`, a debian developer once told me that building from pypi releases is actually discouraged for them.